examples/set-namespace-simple/README.md

```diff
@@ -37,11 +36,9 @@ key-value pair:
 Invoke the function by running the following commands:
 
 ```shell
-$ kpt fn render set-namespace-simple
+$ kpt fn render set-namespace
```

Comment:
`set-namespace-simple` is the package name you fetched from earlier step.
If you change this, the steps here won't work.

Reply:
fixed.